xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -294,13 +302,22 @@ Unlike the <xref:System.Text.Json.Utf8JsonReader>, which is a ref struct, the st
       <Docs>
         <summary>Parses the current JSON token value from the source as a <see cref="T:System.Byte" />.</summary>
         <returns>The value of the UTF-8 encoded token.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+The `GetByte` method throws an exception if the entire UTF-8 encoded token value cannnot be parsed to a <xref:System.Byte> value.
```

Comment:
Typo here too, but please consider mentioning this behavior in the exception summary itself, instead of the remark.
```suggestion
The `GetByte` method throws an exception if the entire UTF-8 encoded token value cannot be parsed to a <xref:System.Byte> value.
```